go

```diff
@@ -149,13 +162,20 @@ def_command :stop_daemons, 'Stop Docker containers running as daemons' do |args|
   end
 end
 
-def_command :rm_containers, 'Remove stopped containers' do |args|
+def_command :rm_containers, 'Remove stopped (non-data) containers' do |args|
   containers_to_stop_regex = "(#{_images(args).join('|')})"
   exec_cmd 'containers=$(docker ps -a | sed -e \'s/.* \([^ ]*$\)/\1/\' | ' \
     "egrep -v '(NAMES|-data)$' | egrep -- '#{containers_to_stop_regex}'); " \
     'if [ ! -z "$containers" ]; then docker rm $containers; fi'
 end
 
+def_command(
+  :rm_all_containers, 'Remove all stopped (non-data) containers') do |args|
+  exec_cmd 'containers=$(docker ps -a | sed -e \'s/.* \([^ ]*$\)/\1/\' | ' \
```

Comment:
could you move `containers=$(docker ps -a | sed -e \'s/.* \([^ ]*$\)/\1/\'` into it's own function or variable. it's used above in 167 too. and maybe the egrep too if possible?